database_backup/database_backup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# For more information on using resources, see https://docs.airplane.dev/reference/env-vars#resources
+DB_HOSTNAME=$(echo $AIRPLANE_RESOURCES | jq -r '.db.host')
+DB_PORT=$(echo $AIRPLANE_RESOURCES | jq -r '.db.port')
+DB_USERNAME=$(echo $AIRPLANE_RESOURCES | jq -r '.db.username')
+DB_NAME=$(echo $AIRPLANE_RESOURCES | jq -r '.db.database')
+DB_PASSWORD=$(echo $AIRPLANE_RESOURCES | jq -r '.db.password')
+
+# If a file name was passed in, use that. Otherwise use the timestamp
+if [ -z "$PARAM_FILE_NAME" ]; then
+    timestamp=$(date +'%Y%m%d_%H%M%S')
+    filename="backup_${timestamp}.zip"
+else
+    filename="$PARAM_FILE_NAME.zip"
+fi
+
+# Create the DB backup
+export PGPASSWORD="$DB_PASSWORD"
+pg_dump -t customers -h $DB_HOSTNAME -p $DB_PORT -U $DB_USERNAME -d $DB_NAME -Fc -f "$filename"
+
+# Copy the database backup to S3
+aws s3 cp ${filename} s3://${AWS_BUCKET}/
+aws s3 ls s3://${AWS_BUCKET}/
+
+data="{\"URL\": \"https://${AWS_BUCKET}.s3.amazonaws.com/${filename}\"}"
+echo "airplane_output_set ${data}"
```

Comment:
You can output specific keys directly:

```suggestion
echo "airplane_output_set:URL \"https://${AWS_BUCKET}.s3.amazonaws.com/${filename}\""
```